apicurito/tools/components/components.go

```diff
@@ -0,0 +1,179 @@
+package components
+
+import (
+
+	"strings"
+	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var Verbs = []string{
```

Comment:
I think this doesn't need to be accessed outside the package

Reply:
Fixed